### DISCUSSION TITLE
Newbie question - how do I reference images/files in my RCL Project from MAUI project?

### DISCUSSION THREAD

**Discussion by bryancass** [dotnet/maui#7552] [Q&A] [answered]
I cannot figure this out and it seems so simple.  I am building a .NET MAUI Blazor Hybrid app so I can share code and components between both mobile and web.

I have three projects in my solution.  I want to place all my icons and images into the RazorClassLibrary project so I can share them between the MAUI project and the Web project.  I reference the RCL Project from the MAUI and Web projects.

For now I'm just using the default MAUI app you get when you create a new .NET MAUI Blazor solution in VS.  I moved the dotnet_bot.svg file from the Resources/Images folder in the MAUI project to the RCL project in the wwwroot/assets/images folder.  So here's the code I'm trying to use to display the image file; this is in the index.razor file in the Pages folder of the RCL.

```
@page "/"

<PageTitle>Index</PageTitle>

<br />
<center><img src="dotnet_bot.svg" /></center>

<h1>Hello, world!!!</h1>

Welcome to your new app.

<SurveyPrompt Title="How is Blazor working for you?" />

<Component1 />
```

I tried all of these to display the image -- none of them work, I just get the broken image icon in the app.
```
<center><img src="images/dotnet_bot.svg" /></center>
<center><img src="/images/dotnet_bot.svg" /></center>
<center><img src="assets/images/dotnet_bot.svg" /></center>
<center><img src="/assets/images/dotnet_bot.svg" /></center>
<center><img src="wwwroot/assets/images/dotnet_bot.svg" /></center>
<center><img src="/wwwroot/assets/images/dotnet_bot.svg" /></center>
```

What am I doing wrong?  Thanks!

**Comment by xtuzy** [comment]
i remember maui will deal with Resources/Images svgs and generate some images, and copy they to platform resources folder when output, i don't use Blazor, i feel you can try use the way of Blazor to set images, then  Include these images as MauiImage at maui .csproj, let maui

**Reply by bryancass** [reply]
I'm sorry, but I really do not understand your post.  How do I display an image on a web page that's stored in a different project of the solution?

**Reply by bryancass** [reply]
I found this repo today - maybe it has some examples of shared resources.  I'll build the app and see what I can learn with it.

https://github.com/microsoft/dotnet-podcasts

**Comment by bryancass** [comment, marked as answer]
OK, I found the answer.  When you reference another project in your solution, you have to use "_content" in the reference, which I guess goes to the solution level to look for the next level.  So in my project, I reference the file in my shared Project.Components project like this:

`<center><img src="_content/Project.Components/assets/images/dotnet_bot.svg" /></center>`

This worked, yay!  Thank you for all your help!
